src/java/picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.java

```diff
@@ -74,12 +73,11 @@
     public boolean REMOVE_DUPLICATES = false;
 
     @Option(shortName = StandardOptionDefinitions.ASSUME_SORTED_SHORT_NAME,
-            doc = "If true, assume that the input file is coordinate sorted even if the header says otherwise.")
-    public boolean ASSUME_SORTED = false;
+            doc = "If not null, assume that the input file has this order even if the header says otherwise.", optional = true)
+    public SAMFileHeader.SortOrder ASSUME_SORT_ORDER = null;
```

Comment:
that works for me.  I am just assuming there are folks who use this option that need some time if they are paying attention to release notes.